extensions-core/histogram/src/main/java/io/druid/query/aggregation/histogram/ApproximateHistogram.java

```diff
@@ -1430,16 +1430,13 @@ public static ApproximateHistogram fromBytesCompact(ByteBuffer buf)
    */
   public static ApproximateHistogram fromBytes(ByteBuffer buf)
   {
-    ByteBuffer copy = buf.asReadOnlyBuffer();
```

Comment:
I don't believe this will work, are there unit tests covering the ApproximateHistogramAggregator that covers reading things from this?  The `fromBytesSparse` and `fromBytesDense` methods both do relative accesses and I would expect eliminating the position setting done in this method to break those.